manifests/site.pp

```diff
@@ -72,7 +72,14 @@
     include prometheus::php_fpm
 }
 
-node 'mwtask1.miraheze.org' {
+# new hardware
+node /^mw(1[012][12])\.miraheze\.org$/ {
```

Comment:
Can this not just be combined with the above with `/^mw(([89]|1[0123])|1[012][12])\.miraheze\.org$/`?